docs/build/html/_static/documentation_options.js

```diff
@@ -1,10 +1,10 @@
 var DOCUMENTATION_OPTIONS = {
     URL_ROOT: document.getElementById("documentation_options").getAttribute('data-url_root'),
-    VERSION: '9.1.0.0',
+    VERSION: '9.1.1.0',
     LANGUAGE: 'None',
     COLLAPSE_INDEX: false,
     FILE_SUFFIX: '.html',
     HAS_SOURCE: true,
     SOURCELINK_SUFFIX: '.txt',
-    NAVIGATION_WITH_KEYS: false
+    NAVIGATION_WITH_KEYS: false,
```

Comment:
Should the trailing "," be here?

Reply:
These are autogenerated files during sphinx build so I wouldn't worry too much about problems in the docs folder that are outside of the docs/source directory